cmd/suggestion/v1alpha2/hyperband/main.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
do you plan to migrate hyperband to python one in another PR? if yes, please move it in this PR